Week1 [Step 2] jinyoo

Rob-Yoo opened the pull request:
@Kioding
제출합니다!

Commit:
Step 2

Commit:
Step 2 - 2nd

Commit:
Step 2 - 3rd

Commit:
Step 2 - 4th

Commit:
Step 2 - 5th

Commit:
docs: 커밋 메시지 템플릿 등록, 순서도 수정

- 커밋 메세지 템플릿을 작성하여 등록하였습니다.
- Step 2의 순서도를 수정하였습니다.

Commit:
fix : Step 2 코드 수정

각 함수들에 매개변수를 지정해서 가독성을 높혔습니다

Kioding commented on CodeStarterCamp_Week1/main.swift:
```diff
@@ -8,21 +8,21 @@
 
-drawIcecream()
-drawStick()
+drawBody(column: 8, row: 11, bodyParticle: "*")
+drawStick(column: 4, stickParticle: "| |")
```
아이스크림의 바디는 `*` 만 쓰고
아이스크림의 막대는 `|  |` 만 쓰는 경우라면 상수로 지정하는 방법도 있겠네요!

지금은 다양한 아이스크림의 형태까지도 고려한 형태네요 👍

Rob-Yoo replied:
앗 그러면 매개변수를 사용하지말고 그냥 바로 print함수에 넣어서 수정하라는 말씀이신가요??

Kioding replied:
정해져있다면 매개변수로 할 필요없이 상수로 넣어도 되고, 다양한 형태를 고려한다면 진유님의 코드가 괜찮은 것 같아요!
저는 단순히 고쳐야한다라는 코멘트였다기보다는, 그런 형태도 있을 수 있다라는 표현이었습니다 🥳

지금의 코드가 더 많은 부분을 담을 수 있어서 수정하지 않아도 괜찮을 것 같아요 👍

Commit:
Docs: README.md 수정

초기 순서도와 나중 순서도의 변화를 담은 리드미 파일 확인

Rob-Yoo committed:
Update README

Rob-Yoo committed:
Update README

Commit:
Docs: README 파일 수정 및 순서도 수정

- 순서도의 박스안에 단어 형태가 아닌 문장 형태로 기입
- 순서도에 출력에 관한 기호 삽입
- README 파일에 초기, 1번째 수정, 2번째 수정 순서로 변화된 순서도 이미지 삽입
